reference/request-bodies/merchant_account_create.yaml

```
@@ -44,3 +44,49 @@ properties:
       requires basic authentication.
     example: super-secret-password
     nullable: true
+
+  visa_network_token_requestor_id:
+    type: string
+    description: |-
+      Requestor ID provided by VISA after onboarding to use Network Tokens.
+    example: e50fa0da-903d-4d54-aacc-4cac57d48df2
+    nullable: true
+
+  visa_network_token_app_id:
+    type: string
+    description: |-
+      Application ID provided by G+D for VISA after onboarding to use Network
```

```suggestion
      Application ID provided for VISA after onboarding to use Network
```
Let's keep G+D out of the docs.

---

Removed in https://github.com/gr4vy/gr4vy-openapi/pull/331/commits/b39dbf62856666ebf85774235cfb19ff5bd8e78b